### PR TITLE
Proposal adjust gasconfig

### PR DESCRIPTION
**Pull request by lyh169**
- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/okex/okexchain/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/okex/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: dev/testnet/testnet.sh [hunk 1]
```diff
@@ -95,7 +95,7 @@ killbyname() {
 init() {
   killbyname ${BIN_NAME}
 
-  (cd ${OKCHAIN_TOP} && make install VenusHeight=1)
+  (cd /Volumes/E/project/github.com/okex/exchain && make install Venus1Height=1 Venus2Height=1 EarthHeight=1 Venus3Height=1)
```
Comment (BananaLF) [inline dev/testnet/testnet.sh]: need use environment

Comment (lyh169) [reply dev/testnet/testnet.sh]: ok

File: libs/cosmos-sdk/baseapp/abci.go [hunk 2]
```diff
@@ -177,6 +182,13 @@ func (app *BaseApp) BeginBlock(req abci.RequestBeginBlock) (res abci.ResponseBeg
 	return res
 }
 
+func (app *BaseApp) UpdateGlobalGasConfig(ctx sdk.Context) {
+	if ctx.IsCheckTx() || ctx.IsTraceTx() {
+		return
+	}
+	stypes.UpdateGlobalGasConfig(app.getGasConfigHandler(app.deliverState.ctx))
```
Comment (BananaLF) [inline libs/cosmos-sdk/baseapp/abci.go]: why not use ctx

File: app/app.go [hunk 3]
```diff
@@ -2,6 +2,7 @@ package app
 
 import (
 	"fmt"
+	stypes "github.com/okex/exchain/libs/cosmos-sdk/store/types"
```
Comment (zjg555543) [inline app/app.go]: adjust import order

File: libs/cosmos-sdk/baseapp/abci.go [hunk 4]
```diff
@@ -3,6 +3,7 @@ package baseapp
 import (
 	"encoding/json"
 	"fmt"
+	stypes "github.com/okex/exchain/libs/cosmos-sdk/store/types"
```
Comment (zjg555543) [inline libs/cosmos-sdk/baseapp/abci.go]: adjust import order

**Review by zjg555543 — COMMENTED** [review]
Can you test from genesis block, to make sure the first block will not smb.

https://github.com/okx/mainnet

yes have verify

File: libs/cosmos-sdk/baseapp/abci.go (hunk 2 above)
Comment (lyh169) [reply libs/cosmos-sdk/baseapp/abci.go]: > why not use ctx

yes， should use ctx

File: libs/cosmos-sdk/baseapp/abci.go (hunk 4 above)
Comment (lyh169) [reply libs/cosmos-sdk/baseapp/abci.go]: > adjust import order

ok

File: app/app.go (hunk 3 above)
Comment (lyh169) [reply app/app.go]: > adjust import order

ok

**Review by BananaLF — DISMISSED** [review]

**Review by zjg555543 — DISMISSED** [review]

**Review by zjg555543 — APPROVED** [review]

**Pull request merged**